reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -801,11 +801,31 @@
 	 * @param <T>      the type of values passing through the {@link Flux}
 	 *
 	 * @return a deferred {@link Flux}
+	 * @see #deferWithContext(Function)
 	 */
 	public static <T> Flux<T> defer(Supplier<? extends Publisher<T>> supplier) {
 		return onAssembly(new FluxDefer<>(supplier));
 	}
 
+	/**
+	 * Lazily supply a {@link Publisher} every time a {@link Subscription} is made on the
```

Comment:
you'd need to describe what the input to the `Function` is (the `Context` attached to said subscriber)

Reply:
Nice one. Added a clarification 👍